shader/antialiasing/cmaa2/cmaa2_common.glsl

```diff
@@ -0,0 +1,196 @@
+// create several presets for defines in shader setup
+#define CMAA_PACK_SINGLE_SAMPLE_EDGE_TO_HALF_WIDTH  1
+#define CMAA2_PROCESS_CANDIDATES_NUM_THREADS        128
+#define CMAA2_DEFERRED_APPLY_NUM_THREADS            32
+#define CMAA2_DEFERRED_APPLY_THREADGROUP_SWAP       1 
+#define CMAA2_USE_HALF_FLOAT_PRECISION              0
+#define CMAA2_SUPPORT_HDR_COLOR_RANGE               0
+#define CMAA2_EDGE_DETECTION_LUMA_PATH              0       // We should use HDR luma from a separate buffer in the future
+#define CMAA_MSAA_SAMPLE_COUNT                      1       // for now force mode without MSAA
+#define CMAA_MSAA_USE_COMPLEXITY_MASK               1
+#define CMAA2_EDGE_UNORM                            0
+#define CMAA2_EXTRA_SHARPNESS                       0
+
+const float c_symmetryCorrectionOffset = 0.22;
+
+#if CMAA2_EXTRA_SHARPNESS
+  const float c_dampeningEffect = 0.11;
+  #define g_CMAA2_LocalContrastAdaptationAmount       0.15f
```

Comment:
why sometimes use `const` and sometimes `#define` ? 

Note: generally `const` is preferred in opengothic, when possible.